frontend/src/serve-citizen/dash-buttons.vue

```diff
@@ -37,28 +37,12 @@
                 :disabled="citizenInvited===true || performingAction || showAdmin"
                 id="add-citizen-button">Back Office</b-button>
     </div>
-    <div>
-      <b-button class="btn-primary mr-1"
-                style="margin-right: 20px"
-                v-if="user.role && (['ANALYTICS', 'GA', 'HELPDESK', 'SUPPORT'].indexOf(user.role.role_code) >= 0)"
-                @click="clickAdmin"
-                id="click-feedback-button">Toggle Admin</b-button>
-      <b-button class="btn-primary mr-1"
-                style="margin-right: 20px"
-                :disabled="showAdmin"
-                @click="clickGAScreen"
-                v-if="user.role && user.role.role_code=='GA'">Toggle GA Panel</b-button>
-      <b-button class="btn-primary"
-                v-if="!showServiceModal"
-                @click="clickFeedback"
-                id="click-feedback-button">Feedback</b-button>
-    </div>
+    <div />
```

Comment:
What is the purpose of this?

Reply:
effectively it creates gaps between the elements that are actually on-screen at a 1:2:1 ratio.  items are flex divs set to "space-between" so its a zero width div that will get a gap on either side of it when without it there would only have been one gap